src/accounts.py

```diff
@@ -0,0 +1,30 @@
+user={"Rhy":"tah"}#key is user, valu is pswd
+accounts = [] #list of users
+
+
+
+def add_account(name,password):
+    print("USER SIGNUP")
+
+    name= str(input("Please enter your name: "))
+    password= str(input("Please enter secret word: "))
+    user=dict()
+
+    if name and password not in user:
+        user['name']=name
```

Comment:
This implementation doesn't align with the expected structure of a user account. The user's password is supposed to be the key whose value is the username. So it is something like `{'password': 'name'}` as opposed to this `{'password': 'password', 'name': 'name'}`. You could implement this by doing `user[password] = name`